package.json

```diff
@@ -30,11 +30,13 @@
     "chai": "^4.2.0",
     "eslint": "^5.6.0",
     "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-jest": "^23.0.4",
```

Comment:
It looks like "eslint-plugin-react-hooks": "^2.1.1", is missing

Reply:
it works without it...